tests/e2e/modify_zoning_labels.sh

```diff
@@ -0,0 +1,197 @@
+# Copyright © 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
```

Comment:
Should we validate SDC connectivity to array(s) as well as part of this script?
That could also come later as part of the next story

Reply:
@JacobGros can we get this information through driver pods or nodes on the cluster?

Reply:
We've looked into this, but the system running the e2e tests will almost assuredly *not* be the worker node and the only way to perform this test would be to SSH into the worker node as a part of the test-- I do not recommend this.